src/utils/test/util.spec.ts

```diff
@@ -155,10 +155,17 @@ describe('util', () => {
       // improperly formatted JSON - note the lack of ':'
       const diagnostic = util.parsePackageJson('{ "someJson" "value"}', mockPackageJsonPath);
 
+      // due to missing colon in input
+      // the text of this message was changed in node version 20
+      const messageText =
```

Comment:
Looks like this is failing CI... I hope we're using the version of Node we say we are 😬